main.go

```diff
@@ -106,6 +106,15 @@ func details(c config.Config, name string) {
 }
 
 func add(c config.Config, path, name string) {
+
+
+	for _, c := range c.Commands {
```

Comment:
Nice! As this is going to be used also for `remove` or `details` commands. I would say to move it inside `config.Config` struct something like:
```
func (c *Config) Get(name string) (Command , error) {
  ....
}
```

So, we can check if that command exists from the outside with:
```
command, err := c.Get(name)
if err != nil {
    fmt.Println("Sorry, this ali is already taken.")
    return
}
```

Reply:
Ok. I'll move it to config.Config